### DISCUSSION TITLE
Installazione striscia Led sotto il letto che si accenda a determinate condizioni

### DISCUSSION THREAD

**Discussion by Valerio0188** [indomus/forum#2701] [GENERALE - Discussioni generiche]
Buongiorno a tutti,

vorrei installare una striscia Led sotto il letto che si accenda alle seguenti condizioni:
1) entro in camera da letto e l'ambiente è buio;
2) non devo essere disteso sul letto (per evitare che si accenda mentre dormo).

Ora, sarebbe facile farlo considerando solo il punto 1) con un sensore di presenza, ma per quanto riguarda la condizione n. 2)?

Avete qualcosa da consigliarmi?

Grazie per l'aiuto.

**Comment by KC-inDomus** [comment]
> Buongiorno a tutti,
> 
> 
> 
> vorrei installare una striscia Led sotto il letto che si accenda alle seguenti condizioni:
> 
> 1) entro in camera da letto e l'ambiente è buio;
> 
> 2) non devo essere disteso sul letto (per evitare che si accenda mentre dormo).
> 
> 
> 
> Ora, sarebbe facile farlo considerando solo il punto 1) con un sensore di presenza, ma per quanto riguarda la condizione n. 2)?
> 
> 
> 
> Avete qualcosa da consigliarmi?
> 
> 
> 
> Grazie per l'aiuto.

https://indomus.it/progetti/dedurre-il-sonno-degli-inquilini-in-una-casa-domotica-con-home-assistant/

**Reply by snakuzzo** [reply]
...oppure dovresti provare con qualche sensore di pressione (tipo quelli che qualcuno usa anche sulle sedie)

**Reply by Valerio0188** [reply]
> ...oppure dovresti provare con qualche sensore di pressione (tipo quelli che qualcuno usa anche sulle sedie)

Non riesco a trovare il sensore di cui parli. Come lo cerco?

**Reply by snakuzzo** [reply]
esempio...

https://community.home-assistant.io/t/diy-zigbee-bed-chair-occupancy-sensor/239517

**Reply by Valerio0188** [reply]
Grazie per lo spunto. Ora che ci penso mi sembra di aver visto tempo fa un video in inglese su YouTube di un "tappetino" da posizionare sotto il materasso integrabile in HA, ma non riesco a trovarlo in vendita su amazon.it. Magari è commercializzato solo sul mercato americano.

**Reply by nromagna** [reply]
Avevo il tuo stesso problema ed ho risolto semplicemente spostando il sendore di movimento in un punto che mi veda solo quando sono in impedi e non sdraito. Nel mio caso il senspre è sopra una mensola sulla testata del letto, posizionandolo vicino al muro la mensola "copre" il raggio d'azione del sensopre e se sono disteso non mi prende, ma appena mi alzo scatta.
Quella del letto è una delle automazioni più usate che ho fatto. Non uso il sensore luce ma vado di orari. Ossia sino alle 21.30 si accende gradualmente sino al 100% luce chiara per una decina di minuti, mentre dalle 22.30 alle 06.00 del mattino si accende colore blu SOLO per 6 secondi. Il tempo di uscire dalla camera, andare in bagno (ho una certa età) e quando rientro in camere si riaccende per altri 6 secondi sempre blu. Non disturba nessuno che dorme visto i pochi secondi, non da fastidio agli occhi in piena notte in quanto blu. TOP

**Reply by nromagna** [reply]
Ovviamete il sensore (questo l'ho fatto in camere dalle bimbe che NON hanno mensole) lo puoi posizionare ai piedi del letto, in modo che scatti solo se "spunta" il piede... ma devi stare attento alla coperte.

**Comment by Valerio0188** [comment]
> Avevo il tuo stesso problema ed ho risolto semplicemente spostando il sendore di movimento in un punto che mi veda solo quando sono in impedi e non sdraito. Nel mio caso il senspre è sopra una mensola sulla testata del letto, posizionandolo vicino al muro la mensola "copre" il raggio d'azione del sensopre e se sono disteso non mi prende, ma appena mi alzo scatta. Quella del letto è una delle automazioni più usate che ho fatto. Non uso il sensore luce ma vado di orari. Ossia sino alle 21.30 si accende gradualmente sino al 100% luce chiara per una decina di minuti, mentre dalle 22.30 alle 06.00 del mattino si accende colore blu SOLO per 6 secondi. Il tempo di uscire dalla camera, andare in bagno (ho una certa età) e quando rientro in camere si riaccende per altri 6 secondi sempre blu. Non disturba nessuno che dorme visto i pochi secondi, non da fastidio agli occhi in piena notte in quanto blu. TOP

Grazie mille per i preziosi consigli.
